users/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+from users.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True, min_length=8, required=True)
+
+    class Meta:
+        model = User
+        fields = ['id', 'email', 'password', 'first_name', 'last_name', 'phone']
+
+    def create(self, validated_data):
+        user = User(
+            email=validated_data["email"],
```

Comment:
No check exists to prevent duplicate emails from being registered. Use `validate_email()` to ensure emails are unique.